services/evaluator/app/infrastructure/api/evaluation_routes.py

```
@@ -0,0 +1,190 @@
+async def get_structured_evaluation(interview_id: str):
+    """Get structured rubric-based evaluation for a specific interview"""
+    from ...infrastructure.llm_provider import load_interview_from_source, get_structured_evaluation
+    
+    try:
+        # Load interview from file (this is what actually exists)
+        interview = load_interview_from_source("file", f"{interview_id}.json")
+        
+        # Get structured evaluation using the rubric schema
+        structured_evaluation = await get_structured_evaluation(interview)
+        
+        return structured_evaluation
```

Ruff (0.13.3)

Consider moving this statement to an `else` block

(TRY300)